docs/pages/web/table.js

```diff
@@ -1599,6 +1599,16 @@ function Example() {
             id="Table.RowExpandable"
           />
         </MainSection.Subsection>
+        <MainSection.Subsection
+          title={generatedDocGen?.TableRowDrawer?.displayName}
+          description={generatedDocGen?.TableRowDrawer?.description}
+        >
+          <GeneratedPropTable
+            generatedDocGen={generatedDocGen.TableRowDrawer}
+            name="Table.RowDrawer"
```

Comment:
`name` and `id` can both also use displayName

Reply:
DONE applied retroactively